### ISSUE TITLE
p-chart SQL update

### ISSUE THREAD

**Issue by saveth** [lzim/teampsd#424]
**BEFORE FILLING OUT THIS TEMPLATE, PLEASE ASSIGN THIS ISSUE TO THE BUG KANBAN UNDER PROJECTS (YES, EVEN IF IT IS A FEATURE. ALL ISSUES START IN THE BUG KANBAN FOR TRIAGE). --------------------->**

**Describe the opportunity**
Updating the p-chart to match baseline EBP metrics and as specified in all the grants.

**FEATURE**

**7. How would you like this feature to work?**
The updated SQL query should:

1. be pulling the data based on the new EBP data definition in #421,
2. use clinic level code instead of facility name,
3. charts should be updated to match new definitions
4. charts can be easily be filtered for any particular clinic of interest 

**8. How urgent is this request? (Urgent/Not Urgent)**
Not urgent

**9. Why is this feature important to you?**
need for R21, R01, and  IIR grants as well as all team tt reports

**10. Is your feature request related to a problem? Please describe.**

**11. Is there a deadline or other scheduled constraint driving your request?"**

**12. Additional context**

**STOP HERE. INFORMATION BELOW IS TO BE FILLED OUT BY WORKGROUPS AS PART OF TRIAGE AND WORK BREAKDOWN PROCESS**

**Estimated person hours to complete:**

**Key people (use @ assignment for people whose input will be necessary):**
@saveth @holbrooa 

**Key dependencies:**
Are there other tasks that need to be completed first?
#421 
**Constraints:**
Is there a resource constraint that may affect the development of the feature?

**Gantt chart needed? Attach link and/or screenshot.**

**Comment by saveth** [comment]
@staceypark @dlkibbe @branscombj  Will changing the p-chart affect any of the guides?

**Comment by staceypark** [comment]
@saveth @dlkibbe @branscombj It won't affect any of the guides but will affect the tt reports.

**Comment by lzim** [comment]
Hi @holbrooa and @saveth 

I wanted to follow-up about the p-charts now that we have omnibus EBP reach, EBPsy reach and EBPharm reach. Is this something this something that Andrew can return to now? 😃 

FYI: @staceypark and @jessfroe

**Comment by staceypark** [comment]
Definitions used for pcharts should be brought up to date with EBP definitions identified for site selection.
These definitions used in the pchart need a level 3 documentation issue in the document_tracker

**Comment by saveth** [comment]
From agenda item ( https://meet.lucidmeetings.com/meeting/225154), still need to be discussed:

1. At what level(s)?
    1. Coverage (anyone with any templates/ prescription)
        -  with AUD
        -  without AUD

    2. Initiation (anyone who starts an EBPsy template, EBPharm Dep follows sail coverage, any AUD, OUD)
        -  with AUD
        -  without AUD
    3. Dose (anyone who complete EBP template or have enough Pharm)
        -  with AUD
        -  without AUD
2. Which EBP measure?
    1. Omnibus EBP
    2. Omnibus EBPsy
    3. Omnibus EBPharm
3. Reference Time:
    1. R21 and R01 refers to the 12months prior to training and 12months post training
    2. IIR refers to the 6 months before and 6months after training

**Comment by staceypark** [comment]
@saveth @holbrooa Your top priority is to finish #462, #615, and #424 as we discussed at the Workgroup Leads meeting yesterday, which we need to submit our protocols prior to launch.

You will both be at the AI conference starting 9/18 so these need to be done by 9/16 or they aren't delayed another 2 weeks.

**Comment by lzim** [comment]
@saveth !

Thanks for these questions.
FYI: @holbrooa and @staceypark 

> From agenda item ( https://meet.lucidmeetings.com/meeting/225154), still need to be discussed:
> 
> 1. At what level(s)?
_EBP Reach R21_
1. P-chart: a) Omnibus EBP Reach (initiate), b) Omnibus EBP Reach (continuity)
2. X-bar chart: Days from intake to first EBP.

_Clinic_
- sta6a 
- for now, let’s not worry about anything at a lower level of aggregation, e.g., a team, as in the past that has been unstable.

>    1. Coverage (anyone with any templates/ prescription)
>       * with AUD
>       * without AUD
- Yes, we need it for all the definitions we have outlined for EBP reach, both with and without AUD.

> 2. Which EBP measure?
>    
>    1. Omnibus EBP
>    2. Omnibus EBPsy
>    3. Omnibus EBPharm

- It would be nice to have evaluate for difference down the road between EBPsy and EBPharm (as well as any differences we may want to explore/report for specific needs: e.g., Dep, PTSD, AUD, OUD), however, we need to report **omnibus EBP Reach**)

> 3. Reference Time:
>    
>    1. R21 and R01 refers to the 12months prior to training and 12months post training
>    2. IIR refers to the 6 months before and 6months after training

- Yes, let’s produce a table of these date ranges for each grant.
- R01/IIR for the purposes of site selection we’re just using CY2018.
- For R21 we have the dates for Clinic 1 and Clinic 2.

**Clinic 1**
- Pre: April 2014-March 2015
- Post: Sept-Aug 2019

**Clinic 2**
- Pre: Oct 2016-Sept 2017
- Post: Oct-2018-Sept 2019

**Comment by saveth** [comment]
@lzim thanks for providing the information regarding what level of EBP measures and what time period you're interested in.

@holbrooa Can you provide information on the following:
1) update on the estimated time to pull the data for this issue
2) what format will the data be in and how it'll look so that I can estimate my time and estimate the type of coding I might need to do to finish my end of this task.

FYI: @staceypark @ritahitching

**Comment by saveth** [comment]
@lzim will we need individual p-chart for each sta6a for the R01/IIR? If so, we will need to finalize our site selections to determine which sites we will need to generate p-charts for.

**Comment by lzim** [comment]
- For @holbrooa’s piece of this: I would rank #615 for the _R01 protocol_ and trial registration - **EBP Reach Dose/Course** higher than #424 p-chart update for the _R21 preliminary findings_

- We need:

- Updated **p-charts*** and **x-bar** charts for the **R21 preliminary findings** manuscript.
- This can be _after_ the details for the **launch of the R01** - we need our trial registration and our protocol manuscripts submitted before we work with clinics in the field.

- If we need to post-pone the p-charts even longer, than we’ll have to do so.

@saveth Yes, we will ultimately need p-charts for _after_ we intervene with the R01/IIR teams (i.e., after s12 of the 12 session plan, in the their final tt3 report), so we are several years away from needing p-charts for each sta6a in the R01/IIR.

**Prioritize: #615
